src/Assert.php

```diff
@@ -817,6 +818,24 @@ public static function uuid($value, $message = '')
         }
     }
 
+    public static function throws(callable $expression, $expected, $message = '')
+    {
+        static::string($expected);
+
+        try {
+            $expression();
+        } catch (Exception $e) {
+            if (is_a($e, $expected, true)) {
```

Comment:
Should we do a `$class === get_class($e)` here? After all, I want a specific exception to be thrown, no? (also requires a test)